### PR TITLE
Add a full_title helper

### PR DESCRIPTION
**Pull request by hoanganhh**

### CODE REVIEW

**Comment by hoanganhh** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
```
Comment (thangtx88) [inline app/helpers/application_helper.rb]: bỏ ()

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,11 @@
...
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (thangtx88) [inline app/helpers/application_helper.rb]: I18n

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (thangtx88) [reply app/helpers/application_helper.rb]: dùng " "  thay cho ' '

**Comment by thangtx88** [comment]
Commented

**Comment by hoanganhh** [comment]
ready

**Comment by thangtx88** [comment]
Merged

**Pull request merged**